test/functional/api/server_requests_spec.lua

```diff
@@ -32,6 +32,24 @@ describe('server -> client', function()
     end)
   end)
 
+  describe('empty string handling', function()
+    -- Checks whether `rpcrequest` can handle a response that contains
+    -- empty strings.
```

Comment:
In this variant comment is rather useless. I expected some addition like “: msgpack parser used to represent second empty string as NULL .ptr”.